packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js

```diff
@@ -144,6 +145,14 @@ export default ({ api, coreSagas }) => {
     }
   }
 
+  const reportBalanceStats = function*() {
+    const btcBalanceR = yield select(selectors.core.data.bitcoin.getBalance)
+    const bchBalanceR = yield select(selectors.core.data.bch.getBalance)
+    const ethBalanceR = yield select(selectors.core.data.ethereum.getBalance)
+
+    yield call(api.incrementCurrencyUsageStats, btcBalanceR.getOrElse(0), ethBalanceR.getOrElse(0), bchBalanceR.getOrElse(0))
```

Comment:
Balance should be 1 or 0 not the actual balance

Reply:
Balance values do not get passed to backend. https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4/src/network/api/wallet/index.js#L223

Reply:
Got it. Another question, why is reportBalanceStats called where it is? Have we fetched this balance data yet?

Reply:
Good question. This depends on the user action of logging in, so that's why it is part of the login routine. The other thought I had was to call this when the top balance component (or balance table) mounts and put the `yield select(balanceSelector)` calls in a saga elsewhere.

Reply:
Balance data gets fetched from the top balance sub-components `cWM` methods. There's a case for putting it there.